resources/security_policy.rb

```diff
@@ -1,14 +1,16 @@
 resource_name :security_policy
 
+default_action :configure
+
 property :policy_template, String, required: false, default: 'C:\Windows\security\templates\chefNewPolicy.inf'
 property :database, String, required: false, default: 'C:\Windows\security\database\chef.sdb'
 property :log_location, String, default: 'C:\Windows\security\logs\chef-secedit.log'
 
 action :configure do
   if node['platform'] == 'windows'
-    template "#{policy_template}" do
+    template '#{policy_template}' do
```

Comment:
Yes, here's the generated secedit command

` execute Secedit /configure /db C:\Windows\security\database\default.sdb /cfg C:\Windows\security\templates\mySecurityPolicy.inf /log C:\Windows\security\logs\chef-secedit.log`

From the following resource

```
security_policy 'Local Policy' do
  policy_template 'C:\Windows\security\templates\mySecurityPolicy.inf'
  database 'C:\Windows\security\database\default.sdb'
  action :configure
end
```

Full output

```       Converging 2 resources
       Recipe: windows-security-policy::template
         * template[C:\Windows\security\templates\mySecurityPolicy.inf] action create (up to date)
       Recipe: windows-security-policy::default
         * security_policy[Local Policy] action configure
           * template[#{policy_template}] action create
             - create new file #{policy_template}
             - update content in file #{policy_template} from none to a507dd
             --- #{policy_template}     2017-06-12 10:37:50.000000000 +0000
             +++ ./chef-#{policy_template}20170612-2380-1qvi0rm 2017-06-12 10:37:50.000000000 +0000
             @@ -1 +1,11 @@
             +[Unicode]
             +Unicode=yes
             +
             +[Version]
             +signature="$CHICAGO$"
             +Revision=1
             +
             +[System Access]
             +
             +[Privilege Rights]
           * execute[Configure security database] action run
             [execute] Completed 1 percent (0/63)       Process Privilege Rights area
       Completed 25 percent (15/63)     Process Privilege Rights area
       Completed 25 percent (15/63)     Process Group Membership area
       Completed 49 percent (30/63)     Process Group Membership area
       Completed 49 percent (30/63)     Process Registry Keys area
       Completed 49 percent (30/63)     Process File Security area
       Completed 49 percent (30/63)     Process Services area
       Completed 65 percent (40/63)     Process Services area
       Completed 73 percent (45/63)     Process Services area
       Completed 73 percent (45/63)     Process Security Policy area
       Completed 77 percent (48/63)     Process Security Policy area
       Completed 84 percent (52/63)     Process Security Policy area
       Completed 88 percent (55/63)     Process Security Policy area
       Completed 93 percent (58/63)     Process Security Policy area
       Completed 100 percent (63/63)    Process Security Policy area


                The task has completed successfully.

                See log C:\Windows\security\logs\chef-secedit.log for detail info.

             - execute Secedit /configure /db C:\Windows\security\database\default.sdb /cfg C:\Windows\security\templates\mySecurityPolicy.inf /log C:\Windows\security\logs\chef-secedit.log


       Running handlers:
       Running handlers complete
       Chef Client finished, 3/4 resources updated in 23 seconds
       Finished converging <default-windows-2012r2> (0m39.64s).
```